File: chapter2-Μπαρμπούτι/ex2.5.py
import random

while True:
    card1 = random.randint(1,11)
    card2 = random.randint(card1+2,13)
    d = card2-card1
    mult = 12/d

    bet = int(input("Πόσο στοιχηματίζεις; "))
    c = int(input("Τί επιλέγεις; "))
    price = bet*mult
    if c>card1 and c<card2:
        print("Κέρδισες ", price)
    else:
        print("Εχασες. Οι αριθμοί ήταν ", card1, card2,
                "και θα κέρδιζες ", price)

File: chapter2-Μπαρμπούτι/ex2.5.rb
while true
    card1 = rand 12
    card2 = card1 + rand(2)+1
    d = card2-card1
    mult = 12/d

    puts("Πόσο στοιχηματίζεις; ")
    bet = gets.chomp.to_i
    puts("Τί επιλέγεις; ")
    c = gets.chomp.to_i
    price = bet*mult
    if c>card1 and c<card2
        puts("Κέρδισες ", price)
    else
        puts("Εχασες. Οι αριθμοί ήταν ", card1, card2,
                "και θα κέρδιζες ", price)
    end
end

